trin-core/src/portalnet/storage.rs

```diff
@@ -708,34 +717,68 @@ impl PortalStorage {
         pool.get()?.execute(CREATE_QUERY, params![])?;
         Ok(pool)
     }
+
+    /// Get a summary of the current state of storage
+    pub fn get_summary_info(&self) -> String {
+        self.metrics.get_summary()
+    }
 }
 
 #[derive(Debug)]
 struct StorageMetrics {
     content_storage_usage_kb: Gauge,
     total_storage_usage_kb: Gauge,
     storage_capacity_kb: Gauge,
+    radius_percent: Gauge,
 }
 
 impl StorageMetrics {
     pub fn new(protocol: &ProtocolId) -> Self {
-        let content_storage_usage_kb = register_gauge!(
+        let content_storage_usage_kb_options = opts!(
             format!("trin_content_storage_usage_kb_{:?}", protocol),
-            "help"
+            "sum of size of individual content stored, in kb"
+        );
+        let (content_storage_usage_kb, registry) = match register_gauge!(
+            content_storage_usage_kb_options.clone()
+        ) {
+            Ok(gauge) => (gauge, Registry::default()),
+            Err(_) => {
+                error!("Failed to register prometheus gauge with default registry, creating new");
+                let custom_registry = Registry::new_custom(None, None)
+                    .expect("Prometheus docs don't explain when it might fail to create a custom registry, so... hopefully never");
+                let gauge = register_gauge_with_registry!(
+                    content_storage_usage_kb_options,
+                    custom_registry
+                )
+                .unwrap();
```

Comment:
Hmmm, this `unwrap()` shouldn't be possible without the linter complaining... Though based off my work in #616 - it seems like the linting process isn't hitting these files... But, either way this unwrap should be handled